spec/requests/api/v1/docs/tags_spec.rb

```diff
@@ -0,0 +1,69 @@
+require "rails_helper"
+require "swagger_helper"
+
+# rubocop:disable RSpec/EmptyExampleGroup
+# rubocop:disable RSpec/VariableName
+
+RSpec.describe "Api::V1::Docs::Tags" do
+  let(:Accept) { "application/vnd.forem.api-v1+json" }
+  let(:api_secret) { create(:api_secret) }
+  let(:user) { api_secret.user }
+
+  describe "GET /tags" do
+    before do
+      user.add_role(:admin)
```

Comment:
I'm not sure if a user needs to be an admin to query thsi information, I don't think they do, but it would be worth double checking that.

Reply:
+1 to removing this role assignment since it's not needed for this endpoint 👍🏼